src/main/web_src/src/components/bestillingsveileder/options/options.js

```diff
@@ -24,18 +24,18 @@ export const BVOptions = ({
 
 	let initialValuesOrganisasjon = {
 		environments: [],
-		organisasjon: {
+		organisasjoner: {
```

Comment:
`organisajon` passer vel kanskje bedre, men `organisasjoner` ble brukt i både backend og forvalter så endret for å få det til å matche.. Skal vi heller bruke organisasjon overalt kanskje? 😄

Reply:
Så lenge det er et organisasjonsobjekt og ikke en array av flere organisasjoner, passer det kanskje best med organisasjon, ja? 😄

Reply:
Ja, godt poeng 😄